packages/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 set(PACKAGE "libyang")
 set(CPP_PACKAGE "libyang-cpp")
-set(PYTHON_PACKAGE "python3-yang")
+set(PYTHON_PACAKGE "python3-libyang")
```

Comment:
Do we want to change the name? Couldn't we keep it and just increase the version or something?

Reply:
Just saw the typo :)

About the name, I'm not sure. I find it confusing. I'll revert the previous name.

Reply:
I meant "confusing" because the python package name is `libyang`.

Reply:
I do not really know so I wanted to discuss it, @rkrejci?

Reply:
I think that since the API is probably different, changing the name makes sense to me.